skimage/color/colorlabel.py

```diff
@@ -108,20 +108,30 @@ def label2rgb(label, image=None, colors=None, alpha=0.3,
         between fully saturated (original RGB, `saturation=1`) and fully
         unsaturated (grayscale, `saturation=0`). Only applies when
         `kind='overlay'`.
+    channel_axis : int, optional
+        This parameter indicates which axis of the output array will correspond
+        to channels. If `image` is provided, this must also match the axis of
+        `image` that corresponds to channels.
+
+        .. versionadded:: 0.19
+           ``channel_axis`` was added in 0.19.
 
     Returns
     -------
-    result : array of float, shape (M, N, 3)
+    result : array of float, shape (..., 3, ...)
```

Comment:
Maybe update line 84 along the same lines: `image : array, shape (M, N) or (..., 3, ...), optional`?

Reply:
done